src/sentry/db/models/manager.py

```diff
@@ -342,6 +342,30 @@ def bind_nodes(self, object_list, *node_names):
             data = node_results.get(node.id) or {}
             node.bind_data(data, ref=node.get_ref(item))
 
+    def snuba_event_from_event_id(self, id_or_event_id, project_id):
```

Comment:
Oh one other thing about this.

This `EventManager` is the manager for the `Event` class (set [here](https://github.com/getsentry/sentry/blob/master/src/sentry/models/event.py#L575) ) but currently `SnubaEvent` doesn't have a manager. So the interface is kinda weird for getting a SnubaEvent, currently it would be

```
Event.objects.snuba_event_from_event_id(...)
```
We could add this same manager as the manager for `SnubaEvent` too, in which case we could use.

```
SnubaEvent.objects.snuba_event_from_event_id(...)
```
But I'm not 100% sure that we should add a django model manager to a class that has nothing to do with django,  also "snuba_event" is kinda redundant in that case. So maybe there should just be a global utility function

```
sentry.event_from_event_id(id_or_event_id, project_id, snuba_event=False)
```
That returns either type of event depending on the param?